src/test/groovy/nebula/plugin/publishing/maven/ResolvedMavenPluginIntSpec.groovy

```diff
@@ -58,6 +58,37 @@ class ResolvedMavenPluginIntSpec extends IntegrationSpec {
         asmDep.version.text() == '2.2.3'
     }
 
+    def 'compatible with nebula.provided-base plugin'() {
+        writeHelloWorld('nebula.hello')
+        buildFile << '''
+            buildscript {
+                repositories { jcenter() }
+                dependencies { classpath "com.netflix.nebula:gradle-extra-configurations-plugin:2.2.0" }
+            }
+            apply plugin: 'java'
+            apply plugin: "nebula.provided-base"
+            apply plugin: 'nebula-maven-publishing'
+            repositories { jcenter() }
```

Comment:
The nebula-test library provides a way to publish and reference libraries locally without having to reach out to JCenter. We will want to do this here. I believe the extra-configurations plugin code has plenty of examples on how to use it.